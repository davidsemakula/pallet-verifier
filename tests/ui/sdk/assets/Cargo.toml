[package]
name = "pallet-assets"
version = "29.1.0"
#authors.workspace = true
#edition.workspace = true
edition = "2021"
license = "Apache-2.0"
#homepage.workspace = true
#repository.workspace = true
description = "FRAME asset management pallet"
readme = "README.md"

#[lints]
#workspace = true

[package.metadata.docs.rs]
targets = ["x86_64-unknown-linux-gnu"]

[dependencies]
codec = { version = "3.6.12", default-features = false, package = "parity-scale-codec" }
impl-trait-for-tuples = "0.2.2"
log = { version = "0.4.22", default-features = false }
scale-info = { version = "2.11.3", default-features = false, features = ["derive"] }
# Needed for various traits. In our case, `OnFinalize`.
sp-runtime = { version = "39.0.0", default-features = false }
# Needed for type-safe access to storage DB.
frame-support = { version = "38.0.0", default-features = false }
# `system` module provides us with all sorts of useful stuff and macros depend on it being around.
frame-system = { version = "38.0.0", default-features = false }
frame-benchmarking = { version = "38.0.0", optional = true, default-features = false }
sp-core = { version = "34.0.0", default-features = false }

[dev-dependencies]
sp-io = { version = "38.0.0", default-features = true }
pallet-balances = { version = "39.0.0", default-features = true }

[features]
default = ["std"]
std = [
	"codec/std",
	"frame-benchmarking?/std",
	"frame-support/std",
	"frame-system/std",
	"log/std",
	"pallet-balances/std",
	"scale-info/std",
	"sp-core/std",
	"sp-io/std",
	"sp-runtime/std",
]
runtime-benchmarks = [
	"frame-benchmarking/runtime-benchmarks",
	"frame-support/runtime-benchmarks",
	"frame-system/runtime-benchmarks",
	"pallet-balances/runtime-benchmarks",
	"sp-runtime/runtime-benchmarks",
]
try-runtime = [
	"frame-support/try-runtime",
	"frame-system/try-runtime",
	"pallet-balances/try-runtime",
	"sp-runtime/try-runtime",
]
