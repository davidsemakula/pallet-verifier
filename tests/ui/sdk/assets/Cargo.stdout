Searching for FRAME pallet definition ...
Searching for dispatchable function definitions ...
Searching for implementations of pallet hooks ...
Searching for public associated function definitions ...
Searching for pallet associated function calls ...
Generating tractable entry points for FRAME pallet ...
Analyzing FRAME pallet with MIRAI ...
Creating summaries for FRAME and Substrate functions ...
Analyzing dispatchables ...
Analyzing dispatchable: `approve_transfer` ...
Analyzing dispatchable: `block` ...
Analyzing dispatchable: `burn` ...
Analyzing dispatchable: `cancel_approval` ...
Analyzing dispatchable: `clear_metadata` ...
Analyzing dispatchable: `create` ...
Analyzing dispatchable: `destroy_accounts` ...
Analyzing dispatchable: `destroy_approvals` ...
Analyzing dispatchable: `finish_destroy` ...
Analyzing dispatchable: `force_asset_status` ...
Analyzing dispatchable: `force_cancel_approval` ...
Analyzing dispatchable: `force_clear_metadata` ...
Analyzing dispatchable: `force_create` ...
Analyzing dispatchable: `force_set_metadata` ...
Analyzing dispatchable: `force_transfer` ...
Analyzing dispatchable: `freeze` ...
Analyzing dispatchable: `freeze_asset` ...
Analyzing dispatchable: `mint` ...
Analyzing dispatchable: `refund` ...
Analyzing dispatchable: `refund_other` ...
Analyzing dispatchable: `set_metadata` ...
Analyzing dispatchable: `set_min_balance` ...
Analyzing dispatchable: `set_team` ...
Analyzing dispatchable: `start_destroy` ...
Analyzing dispatchable: `thaw` ...
Analyzing dispatchable: `thaw_asset` ...
Analyzing dispatchable: `touch` ...
Analyzing dispatchable: `touch_other` ...
Analyzing dispatchable: `transfer` ...
Analyzing dispatchable: `transfer_all` ...
Analyzing dispatchable: `transfer_approved` ...
Analyzing dispatchable: `transfer_keep_alive` ...
Analyzing dispatchable: `transfer_ownership` ...
Analyzing pub assoc fns ...
Analyzing pub assoc fn: `account_balances` ...
Analyzing pub assoc fn: `adjust_extra` ...
Analyzing pub assoc fn: `balance` ...
Analyzing pub assoc fn: `maybe_balance` ...
Analyzing pub assoc fn: `maybe_total_supply` ...
Analyzing pub assoc fn: `total_supply` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Balanced>::done_deposit` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Balanced>::done_withdraw` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Create>::create` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Destroy>::destroy_accounts` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Destroy>::destroy_approvals` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Destroy>::finish_destroy` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Destroy>::start_destroy` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Inspect>::asset_exists` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Inspect>::balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Inspect>::can_deposit` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Inspect>::can_withdraw` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Inspect>::minimum_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Inspect>::reducible_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Inspect>::total_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Inspect>::total_issuance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::InspectEnumerable>::asset_ids` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Mutate>::done_burn_from` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Mutate>::done_mint_into` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Mutate>::done_transfer` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Refund>::deposit_held` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Refund>::refund` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Unbalanced>::decrease_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Unbalanced>::handle_dust` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Unbalanced>::handle_raw_dust` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Unbalanced>::increase_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Unbalanced>::set_total_issuance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::Unbalanced>::write_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::approvals::Inspect>::allowance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::approvals::Mutate>::approve` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::approvals::Mutate>::transfer_from` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::metadata::Inspect>::decimals` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::metadata::Inspect>::name` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::metadata::Inspect>::symbol` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::metadata::MetadataDeposit>::calc_metadata_deposit` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::metadata::Mutate>::set` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::roles::Inspect>::admin` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::roles::Inspect>::freezer` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::roles::Inspect>::issuer` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::roles::Inspect>::owner` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungibles::roles::ResetTeam>::reset_team` ...
